internal/utils/config.go

```diff
@@ -329,6 +331,8 @@ func LoadConfigFS(fsys afero.Fs) error {
 			KongId = "supabase_kong_" + Config.ProjectId
 			GotrueId = "supabase_auth_" + Config.ProjectId
 			InbucketId = "supabase_inbucket_" + Config.ProjectId
+			InbucketConfigVolumeId = "supabase_inbucket_config_" + Config.ProjectId
```

Comment:
What's the use case for persisting the config volume? There's only a single file in `/config/greeting.html` which is created by default. Other configs should be added via [env vars](https://github.com/inbucket/inbucket/blob/main/doc/config.md) instead.